src/containers/Explorer/FilterNav.js

```diff
@@ -59,22 +118,24 @@ const FilterNavComponent = props => {
         </Text>
         <Checkbox
           label="Active"
-          onChange={() => toggleStageFilter('active')}
+          onChange={() => {
+            toggleStageFilterAction('active');
```

Comment:
We're using semicolon in these newline'd blocks, and not for the one-liners - this is something that will be more uniform and clean with standardjs formatting.

Reply:
@codeluggage in general this has been convention - no semicolon needed on the () => toggleStageFilter format - it's because the one liner includes return behind the scenes. I actually think a semicolon breaks it.